api/secret.go

```diff
@@ -28,6 +28,7 @@ type Secret struct {
 // SecretAuth is the structure containing auth information if we have it.
 type SecretAuth struct {
 	ClientToken string            `json:"client_token"`
+	AccessorID  string            `json:"accessor_id"`
```

Comment:
I know this is going to be an annoying code change, but...I don't like "AccessorID". I used "Accessor" in the RFC because it's a noun; calling it an AccessorID means that it's an ID of the accessor, which is not what it is. "AccessID" could work, but I preferred Accessor over AccessID because AccessID could be confused as the actual important value (a value to "access Vault").